analysis/config/config.go

```diff
@@ -371,6 +367,32 @@ func Load(filename string) (*Config, error) {
 	return cfg, nil
 }
 
+// LoadEscape adds the escape configuration settings from escapeConfigBytes into c.
+func LoadEscape(c *Config, escapeConfigBytes []byte) error {
```

Comment:
It's a little counterintuitive to have the user of the package manage the loading of different parts of the configuration. Does the user have to retrieve the escape config file info, load it, and then call this function which will do the check again before using the bytes? 
Can we provide a function that does all the loading?

Reply:
`escapeConfigBytes` is the byte representation of the escape-config.json file. Used here: https://github.com/awslabs/ar-go-tools/pull/66/files#diff-8342ad4f71f577e5db7687522a945f15802f375a047ea3100c11d3ef2dfb442dR115

I can update the godoc to say this explicitly if that helps.

Reply:
The idea is that `LoadX` functions shouldn't perform any I/O: the I/O should be the caller's responsibility. Maybe changing the type from `[]byte` to `io.Reader` could make this more obvious but I figured `[]byte` was the simpler option.

Reply:
Maybe we can then add something in the return value of LoadConfig that tells the caller whether they need to do another LoadX? It should also not be the caller's responsibility to figure out the logic to load a complete config.

What I was proposing is another function that does I/O and is explicit about it. LoadConfigFromFile?

Reply:
True, maybe `config.Load` can take the config bytes and escape config bytes as arguments which would eliminate the need to have the `config.LoadEscape` function. Ideally the escape config would be part of the regular config file instead of a separate file.

`config.LoadGlobal` already performs file I/O. Granted, it is a special case but I'm not sure if we need a general-purpose `LoadConfigFromFile` function?

Reply:
If we don't have a central place for the file I/O logic for loading a complete config, maintaining the cli which is harder to test will be difficult. I don't know if the cli was tested with the new functions (does the config reloading from disk work as expected?).